presto-hive-metastore/pom.xml

```diff
@@ -101,6 +107,34 @@
             <artifactId>aws-java-sdk-sts</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>dynamodb</artifactId>
+            <version>2.17.131</version>
```

Comment:
Presto currently makes use of AWS SDK 1.x. I think we should use the same version here as well otherwise it could result in conflicts and difficulty to maintain. We can migrate to AWS SDK 2.x collectively as a project once it is prioritized.

Reply:
So the idea behind introducing 2.x dependencies was latest iceberg code [1.3.1] is making use of 2.x sdk. So adding 1.x version for this dependency might have resulted in issues. Moreover, Trino also currently has both the dependencies, they are adding the 2.x dependencies as bom. So I don't think it should create a major issue. What do you think? 
If you still strongly feel we should proceed with 1.x only for now, I can rework these imports. Let me know either ways.